apps/wallet-ui/biome.json

```diff
@@ -2,6 +2,6 @@
   "$schema": "https://biomejs.dev/schemas/1.8.1/schema.json",
   "extends": ["../../biome.json"],
   "files": {
-    "ignore": []
+    "ignore": ["./src/components/ui/**/*"]
```

Comment:
why?

Reply:
shad components do funky things that biome doesn't agree with, and this felt better than littering those files with ignore comments

Reply:
just format them once